frontend/views/utils/translations.js

```diff
@@ -17,6 +17,24 @@ i18next.use(XHR).init({
 Vue.prototype.L = L
 Vue.prototype.LTags = LTags
 
+/*
+Examples:
+
+Simple string:
+  i18n Hello world
+
+String with variables:
+  i18n(
+    :args='{ name: ourUsername }'
+  ) Hello {name}!
+
+String with markup inside:
+  i18n(
+    compile
+    :args='{ ...LTags("strong"), name: ourUsername }'
+  ) Hello {strong_}{name}{_strong}!
```

Comment:
Maybe a different example for this one, because if {name} is a variable too, you should instead just do:

```
  i18n(
    :args='{ name: `<strong>${ourUsername}</strong>`` }'
  ) Hello {name}!
```

Note also `compile` should only be added for strings that use Vue.js components like `<router-link>`, otherwise it shouldn't be added. Regular HTML works just fine without it.

In this case the example can be:

```
  i18n(
    :args='LTags("strong")'
  ) Hello and {strong_}welcome{_strong}!
```

Reply:
done, added more examples!